src/app/clusters/general_diagnostics_server/general_diagnostics_server.cpp

```diff
@@ -29,7 +30,7 @@ using namespace chip::app;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::GeneralDiagnostics::Attributes;
 using chip::DeviceLayer::ConnectivityMgr;
-using chip::DeviceLayer::PlatformManager;
+using chip::DeviceLayer::PlatformMgr;
```

Comment:
Why this change?  Why not `using` both and avoid the `DeviceLayer` prefixes below?